src/components/GameCard.tsx

```diff
@@ -9,17 +9,71 @@ interface GameCardProps {
     gameTitle: string;
     gameImage: any;
     gameDescription: string;
+    gamePlatforms: Array<string>;
+    gameGenres: Array<string>;
+    gameUniverses: Array<string>;
+    gameCompanies: Array<string>;
+    gameReleaseDate: Array<string>;
+    gameRating?: number;
+    gameReview?: string;
+    gameRanking?: string;
+    yearPlayed?: number;
+    yearReceived?: number;
+    notes?: string;
+    platformsOwnedOn?: Array<string>;
+    dateAdded?: String;
+
 }
 
-export function GameCard({gameTitle, gameImage, gameDescription } : GameCardProps ) {
+export function GameCard(gameCardProps : GameCardProps ) {
     // will eventually be an API function
     const addToLibrary = () => true;
 
     return (
         <div>        
-            <h1>{gameTitle}</h1>
-            <svg>{gameImage}</svg>
-            <p>{gameDescription}</p>
+            <h1>{gameCardProps.gameTitle}</h1>
+            <img 
+            src = {gameCardProps.gameImage[0]}
+            alt = "Game"
```

Comment:
💯 we should probably think more about accessibility on the frontend as we go forward